packages/bls-kms-local/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@sphereon/ssi-sdk-bls-key-manager",
+  "description": "Sphereon KMS implementation that provides BLS crypto",
```

Comment:
Not sure this description really reflects what this plugin is about? it is about signing?

Reply:
Changed the description, but we need to support bls keys not only in the key store but in did:key provider and resolver as well. Veramo has a partial implementation of it nowadays, so I'm not sure if I will create a PR for them or just wait until their changes are pushed to the upstream.